addons/website_mail_channel/views/snippets.xml

```diff
@@ -44,7 +44,13 @@
         <div data-js='subscribe'
              data-selector=".js_follow_alias"
              data-drop-near="p, h1, h2, h3, blockquote, .card">
-            <we-button data-select_mailing_list="" data-no-preview="true">Change Discussion List</we-button>
+            <we-row>
+                <we-select class="select_discussion_list" data-attribute-name="id" data-no-preview="true">
+                    <!-- 'we-button' added programmatically with DB data -->
+                </we-select>
+                <we-button class="fa fa-fw fa-plus" title="Create a public discussion group in your backend"
+                           data-create-channel="" data-no-preview="true"/>
+            </we-row>
```

Comment:
There is no label for the select/row ?

Reply:
Nop, by design/spec, was discussed with sbu.
Does the UX seems weird to you or is it just codewise?

PS: Another `title` attribute :D

Reply:
I did not look at the UX ;) It was just to be sure it was not an oversight